website/src/examples/dropbox.md

```diff
@@ -1,9 +1,94 @@
----
-title: Dropbox
-type: examples
-order: 1
----
+<!DOCTYPE html>
```

Comment:
Since this uses a layout, we already have `<html>` and page styling. You can rename the file to `dropbox.html` remove everything above `<h1>`, and starting/below `<body>`